docs/source/usage/index.rst

```diff
@@ -0,0 +1,81 @@
+.. _usage:
+
+Usage
+=====
+
+This is a walkthrough of the CellMincer pipeline using an Optosynth dataset as an example. This dataset (2.4 GB) can be `downloaded here <https://storage.cloud.google.com/bw-cellmincer-dev/Optosynth/raw/optosynth__1__20__50.tif>`_.
```

Comment:
In this section, we present a practical quick-start tutorial for using CellMincer using a synthetic Optosynth dataset as an example.